pkg/config/config_template.yaml

```diff
@@ -154,12 +160,14 @@ metadata_collectors:
 #     template_url: 127.0.0.1
 #     username:
 #     password:
-
+{{ end -}}
+{{- if .Logging }}
```

Comment:
Why isn't logging in common?

Reply:
see https://github.com/DataDog/datadog-agent/pull/799#discussion_r150495530